t/tick.html

```diff
@@ -4,9 +4,12 @@
 
 <input type="checkbox" name="foo" value="hello" /> Hello<br />
 <input type="checkbox" name="foo" value="bye"   /> Bye<br />
-<input type="checkbox" name="foo" value="arse" /> Arse<br />
+<input type="checkbox" name="foo" value="parse" /> Parse<br />
```

Comment:
😆

Reply:
Not a change I would have made, but I'll approve 😆